nbs/examples/MLFlow_and_neuralforecast.ipynb

```diff
@@ -0,0 +1,428 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #8.    from mlflow.client import MlflowClient
</pre><p>please move this import above</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Moved import above as suggested.</p>